gu-provider/src/hdman.rs

```diff
@@ -533,40 +517,6 @@ impl Handler<DestroySession> for HdMan {
     }
 }
 
-/// HdMan Errors
-// impl note: can not use error_chain bc it does not support SerDe
-#[derive(Serialize, Deserialize, Debug)]
-pub enum Error {
-    Error(String),
-    IoError(String),
-    NoSuchSession(String),
-    NoSuchChild(String),
-}
-
-impl From<io::Error> for Error {
-    fn from(e: io::Error) -> Self {
-        Error::IoError(e.to_string())
-    }
-}
-
-impl fmt::Display for Error {
-    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
-        match self {
-            Error::Error(msg) => write!(f, "error: {}", msg)?,
-            Error::IoError(msg) => write!(f, "IO error: {}", msg)?,
-            Error::NoSuchSession(msg) => write!(f, "session not found: {}", msg)?,
-            Error::NoSuchChild(msg) => write!(f, "child not found: {}", msg)?,
-        }
-        Ok(())
-    }
-}
-
-impl From<String> for Error {
-    fn from(msg: String) -> Self {
-        Error::Error(msg)
-    }
-}
-
 impl Handler<status::GetEnvStatus> for HdMan {
```

Comment:
shouldn't `GetEnvStatus` be also part of  `gu-envman-api` ?

Reply:
This message is not accessible to RPC.

Reply:
why?

Reply:
this is a callback for a different interface.